### DISCUSSION TITLE
How to enable a page like the Docusaurus showcase?

### DISCUSSION THREAD

**Discussion by ghost** [facebook/docusaurus#9625] [Q&A]
I'm using Docusaurus for a personal portfolio site, and I think the Docusaurus [showcase](https://docusaurus.io/showcase) UI would be ideal for a home page that nicely shows a table of cards, one for each project, with tags and search at the top. I would love to use this UI if it's available. 

Does anyone know a way to do this?

I searched around and found:
- this [Canny feedback post](https://docusaurus.io/feature-requests/p/showcase-page-as-a-plugin), asking for the showcase in plugin for.
- this [GitHub issue](https://github.com/facebook/docusaurus/issues/6882) (linked in the reply to the Canny post from @Josh-Cena)
- this [plugin](https://www.jsdelivr.com/package/npm/docusaurus-plugin-showcase) (which oddly is not listed in the [community plugin directory](https://docusaurus.community/plugindirectory/) or [list of community plugins](https://docusaurus.io/community/resources#community-plugins) in the Awesome Resources part of the Docusaurus docs)

I would very much appreciate any tips on how to do this! Thanks.

**Comment by netraular** [comment]
Any updates on how can it be implemented?

**Reply by slorber** [reply]
We plan to make it a first-class plugin since a few sites copied our code: https://github.com/facebook/docusaurus/pull/9967

The PR is working but is not ready to be merged, I plan to continue it once I finish other things I'm working on
